### DISCUSSION TITLE
OSL Documentation

### DISCUSSION THREAD

**Discussion by mprater** [imageworks/OpenShadingLanguage#1324] [General]
Instead of using a separate github repo for staging the new documentation files, we'll be incorporating them into the master branch instead. This will keep the licensing consistent with the OSL project itself and avoid any cumbersome issues that might arise from porting them from another repo.

These new files will be staged in a new `doc` directory.

There isn't much to start with, but... baby steps.

**Comment by fpsunflower** [comment]
Makes sense. I would definitely encourage small incremental PRs instead of trying to collect a tons of content "offline". It will make it easier for us to proofread the content as its written and suggest any course corrections before you've gone too far down a particular path.
